packages/polkadart_scale_codec/test/util/functions_test.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:polkadart_scale_codec/src/util/utils.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+void main() {
+  group('functions checkSignedInt()', () {
+    test('Should returns normally when bitSize is 8', () {
```

Comment:
Many ways to improve this :)
```suggestion
    test('Given bitSize is 8 and value is within range check should pass', () {
```

```suggestion
    test('Given bitSize is valid and value is within range check should not throw', () {
```

```suggestion
    test('Given bitSize is 8 and value is valid check should not fail', () {
```